pandas/core/strings.py

```diff
@@ -912,7 +915,7 @@ def str_extract(arr, pat, flags=0, expand=True):
         return _str_extract_frame(arr._orig, pat, flags=flags)
     else:
         result, name = _str_extract_noexpand(arr._parent, pat, flags=flags)
-        return arr._wrap_result(result, name=name, expand=expand)
+        return arr._wrap_result(result, name=name, expand=expand, returns_string=True)
```

Comment:
umm, what is the purpose of this?

Reply:
In general, `StringMethods._wrap_result` is used to return string-like (`.str.upper`) and numeric (`.str.contains`). We need to avoid trying to construct a StringArray for the numeric ones. Will leave a comment.

This specific one can be removed (I think I changed the default from False to True at some point).